### ISSUE TITLE
Request for Touch ID always pops up, even after "cancel", due to focus on database manager

### ISSUE THREAD

**Issue by svn2208** [strongbox-password-safe/Strongbox#613]
Strongbox Version: 1.17.0
Mac OS: 12.3

When I activate _Lunch at Startup_ to my fav kdbx in the Database Manager and start using my MacOS I run into stituation, e.g. switching desktops, when Strongbox ask to unlock my kdbx with the Touch ID popup (because, the DB Manager gains focus). When I press cancel, it will show up again, because the Database Manager gains focus again and try to unlock the kdbx. 
Evertime when the database manager gains focus, the touchid popup apears. 
This is pretty anoying and can only be prevented, when I deactivate _Launch at startup_

I am not sure, if this is intendat or an unwanted behaviour.

**Comment by tomthorogood** [comment]
This happens to me also. I generally use everything in full-screen (zoomed) mode on OSX, and use the three-finger swipe to move between applications. Whenever I swipe past Strongbox, the touchID request appears. It is very annoying. the only way to stop it is to either unlock the safe or quit strongbox.

**Comment by dan-el** [comment]
@strongbox-mark I shared a screen recording of the behaviour via iCloud. For me it only happens if Strongbox is "on top" on the desktop "swipe by" using the 3-finger-gesture.

**Comment by strongbox-mark** [comment]
Yes, a quick fix for this should be to disable "Auto Prompt on Activate" checkbox on the Lock Screen. Does that work for you? @dan-el Could you send a link to the screen recording?

**Comment by dan-el** [comment]
@strongbox-mark Here is the link: https://www.icloud.com/iclouddrive/0eb87AvBiBb-ufPz6Zujf7brA#screen_recording

**Comment by strongbox-mark** [comment]
Thanks @dan-el - This behaviour looks to me correct if you have the "Auto Prompt on Activate" feature on. Right? I'm not seeing the bug here.

**Comment by dan-el** [comment]
Hi @strongbox-mark ,
I'd agree, but the thing is that the TouchID prompt stays on top on all desktops, not only the one that the Strongbox app is on. That behaviour is not on visible on the screen recording unfortunately as I always cancelled the prompt immediately, but if I had swiped to another Desktop without cancelling the prompt would have been there too.
I think that is the annoying part @svn2208 is referring to.

**Comment by strongbox-mark** [comment]
OK, thanks, I'm not sure there's a way to limit this Touch ID dialog/request to a desktop, in fact I'm fairly sure there isn't... I think the solution in this case would be to simply turn off the "Auto-Prompt" switch.

**Comment by strongbox-mark** [comment]
Going to close this one as the solution is pretty clear.
